rocket/rocket.py

```diff
@@ -73,8 +73,8 @@ def launch(
         if not os.path.exists(f"{home}/.databrickscfg"):
             raise Exception("Databricks cli not configured. Run `databricks configure --token`.")
 
-        if os.getenv("DATABRICKS_TOKEN") is None:
-            raise Exception("DATABRICKS_TOKEN must be set for db-rocket to work")
+        if os.getenv("DATABRICKS_TOKEN"):
+            print("Note: DATABRICKS_TOKEN is set, it could override the token in ~/.databrickscfg and cause errors.")
```

Comment:
I actually do set `DATABRICKS_TOKEN` in my environment rather than use `databricks configure` -- I wonder how common this is. I don't think it changes much either way though.

Reply:
I think we should recommend 1 way, so as to keep it simple as one can affect the other and cause debugging problems.  I see developers putting secrets on startup files and I think ideally we desincourage that direction as leaves the laptop leaks more serious. So my opinion is to leave this to be managed by databricks (even though actually its just stored plain somewhere else atm).  Ideally databricks should provide 2factor authentication, or GYG.